subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/api/ConfigurationRoleUsageIntegrationTest.groovy

```diff
@@ -109,7 +108,7 @@ class ConfigurationRoleUsageIntegrationTest extends AbstractIntegrationSpec {
         fails 'help'
 
         and:
-        assertUsageLockedFailure(configuration)
+        assertUsageLockedFailure(configuration, 'Intended Consumable')
```

Comment:
The test name implies that the configuration under test is roleless. However, it is no longer roleless.

Reply:
Good catch, I'll move this to the proper region.